internal/fs/fs.go

```diff
@@ -988,7 +988,7 @@ func (fs *fileSystem) syncFile(
 	// syncFile can be triggered for unlinked files if the fileHandle is open by same
 	// or another user. Hence, returning an error when syncFile is called for local file.
```

Comment:
Hence, ignoring the syncFile call for local file.

Reply:
fixed.